schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INT,
```

Comment:
[OPTIONAL] You can make the id column not null and autoincremented, or also mark the id column as primary key

Reply:
Thanks for the approval, and for the suggestion.